Add file size limit on source maps upload (#8837)
DataDog/documentation: content/en/real_user_monitoring/guide/upload-javascript-source-maps.md

If your front-end Javascript source code is minified, you will need to upload your source maps to Datadog so that we are able to deobfuscate your different stack traces. For a given error, you will then get access to the file path, the line number, as well as a code snippet for each frame of the related stack trace.

## Instrument your code

> You must configure your Javascript bundler so that, when minifying your source code, it generates source maps which directly include the related source code in the `sourcesContent` attribute. In addition, you should make sure that the size of each source map augmented with the size of the related minified file does not exceed __our limit of 50mb__. Check below some configurations for popular Javascript bundlers.

Reviewer: ```suggestion
You must configure your Javascript bundler so that, when minifying your source code, it generates source maps which directly include the related source code in the `sourcesContent` attribute. In addition, ensure that the size of each source map augmented with the size of the related minified file does not exceed __our limit of 50mb__. Check below for some configurations for popular Javascript bundlers.
```

        javascript.464388.js.map
```

> <div class="alert alert-info">If, for example, the sum of the file size of <code>javascript.364758.min.js</code> and <code>javascript.364758.js.map</code> exceeds __our 50mb limit__, you could reduce it by configuring your bundler to split the source code into multiple smaller chunks ([see how yo do it with WebpackJS][https://webpack.js.org/guides/code-splitting/]).</div>

Reviewer: ```suggestion
<div class="alert alert-info">If, for example, the sum of the file size of <code>javascript.364758.min.js</code> and <code>javascript.364758.js.map</code> exceeds __the 50mb limit__, reduce it by configuring your bundler to split the source code into multiple smaller chunks ([see how to do this with WebpackJS][https://webpack.js.org/guides/code-splitting/]).</div>
```
